app/scripts/lib/metrics.js

```diff
@@ -364,6 +367,27 @@ define(function (require, exports, module) {
       // The idenifier is kept to avoid updating all metrics queries.
       return 'screen.' + viewName;
     },
+    /**
+     * Log when an experiment is shown to the user
+     *
+     * @param {String} choice - type of experiment
+     * @param {String} group - the experiment group (treatment or control)
+     */
+
```

Comment:
good doc string 👍 , remove the line break between the docstring and the function name though